src/Datadog.Trace/Tagging/HttpTags.cs

```diff
@@ -4,15 +4,25 @@ namespace Datadog.Trace.Tagging
 {
     internal class HttpTags : InstrumentationTags, IHasStatusCode
     {
-        protected static readonly IProperty<string>[] HttpTagsProperties =
-            InstrumentationTagsProperties.Concat(
-                new Property<HttpTags, string>(Trace.Tags.HttpStatusCode, t => t.HttpStatusCode, (t, v) => t.HttpStatusCode = v),
-                new Property<HttpTags, string>(HttpClientHandlerTypeKey, t => t.HttpClientHandlerType, (t, v) => t.HttpClientHandlerType = v),
-                new Property<HttpTags, string>(Trace.Tags.HttpMethod, t => t.HttpMethod, (t, v) => t.HttpMethod = v),
-                new Property<HttpTags, string>(Trace.Tags.HttpUrl, t => t.HttpUrl, (t, v) => t.HttpUrl = v),
-                new Property<HttpTags, string>(Trace.Tags.InstrumentationName, t => t.InstrumentationName, (t, v) => t.InstrumentationName = v));
+        private readonly HttpTagsKeysMapping _convension;
 
-        private const string HttpClientHandlerTypeKey = "http-client-handler-type";
+        public HttpTags(HttpTagsKeysMapping convension)
```

Comment:
`convension` should be `convention`

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/63/commits/115d77b78c97553d89e6122bd86071e8d7328945